tests/csm/rest/test_capacity_usage.py

```diff
@@ -263,14 +233,14 @@ def test_33899(self):
 
         self.log.info("[START] Failure loop")
         for failure_cnt in range(1, self.kvalue + 2):
```

Comment:
what is the significance of + 2 ? if it is fixed, can we add to const ?

Reply:
It is fixed to 2 as we want to create one more than K value,